src/pages/ChangeSet.tsx

```diff
@@ -0,0 +1,36 @@
+import { Box, Button, Toolbar, Typography } from '@mui/material';
+import PageContainer from 'components/shared/PageContainer';
+import { FormattedMessage } from 'react-intl';
+import ChangeSetTable from '../tables/ChangeSet';
+
+const ChangeSet = () => {
+    return (
+        <PageContainer>
+            <Toolbar sx={{ justifyContent: 'space-between' }}>
+                <Typography>
+                    <FormattedMessage id="changeSet.header" />
+                </Typography>
+
+                <Box>
+                    <Button
+                        /* TODO: Differentiate bkg color of delete button */
+                        variant="contained"
+                        disabled={true}
```

Comment:
Depending on our current linting rules.... but I think for optional boolean props like this you can just include `disabled` and don't need the true value.

Reply:
**PENDING CORRECTION.** Unless the updated linting effected things, this is 100% possible and preferred. The running disclaimer applies here as well. I arrived at that conclusion shortly after opening this PR. It was meant to be a draft to review and correct any mistakes before anyone else took a critical look, but I forget to press that button yesterday evening. The plan was to make it a draft and leave behind comments indicating pending changes before you got to it today, haha but it looks like you logged on last night.